src/briefcase/integrations/cookiecutter.py

```diff
@@ -77,6 +77,28 @@ def escape_non_ascii(obj):
         environment.filters["escape_non_ascii"] = escape_non_ascii
 
 
+class GradleEscape(Extension):
+    """Jinja2 extension to escape strings for Gradle as well."""
+
+    def  __init__(self, environment):
```

Comment:
This has been picked up by the pre-commit handler; there's an extra space here:
```suggestion
    def __init__(self, environment):
```